src/components/PeoplesTable/PeopleTable.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Person } from '../../types';
+import { PersonInfo } from '../PersonInfo';
+
+type Props = {
+  people: Person[]
+  selectedSlug:string,
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, selectedSlug }) => {
+  const findPersonByName = (name:string | null) => {
+    return people.find(parent => parent.name === name);
+  };
```

Comment:
do you need this function here? if it need to be here only for passing as a props to the other component - it's not the best way of it's usage
let's move it somewhere in separate file
or it will be better to add `mother` and `father` properties for each person after retrieving them from the server